### PR TITLE
Bugfix/manage empty list attribute

### LINKED ISSUE #78: Cannot parse empty list as Link
**Describe the bug**

The parser is not able to parse empty lists `[]` as a `Link` attribute.

For example, the `aws_security_group` component has a `Link` attribute named `vpc_id`.

* This file is parsable (no empty list):
    ```terraform
    resource "aws_security_group" "aws_security_group_1" {
        vpc_id = ["aws_vpc_1"]
    }

    resource "aws_vpc" "aws_vpc_1" {
    }
    ```

* But this one is not (empty list) :
    ```
    resource "aws_security_group" "aws_security_group_1" {
        vpc_id = []
    }

    resource "aws_vpc" "aws_vpc_1" {
    }
    ```

There are no errors in the console.

**To Reproduce**

https://github.com/ditrit/terrator-plugin/assets/20184007/43e4762d-d97e-43cb-8e3a-2ad9a6da35e6

**Application**

- Terrator-plugin commit hash: `33df373a87467354227d707a60b1dd34b4cb9ad1`

### PR DESCRIPTION
**Pull request by k-y0u**
resolves #78

### CODE REVIEW

**Review by viennem — DISMISSED** [review]
I confirm that the fix works.
I did not review the code.

File: tests/resources/tf/bug78_emptyListAttribute.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+resource "aws_security_group" "aws_security_group_1" {
+    vpc_id = []
+}
+
+resource "aws_vpc" "aws_vpc_1" {
+}
```
Comment (Zorin95670) [inline tests/resources/tf/bug78_emptyListAttribute.tf:6]: missing empty line at the end of file

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Review by justinehell — CHANGES_REQUESTED** [review]

File: tests/resources/tf/bug78_emptyListAttribute.tf (hunk 1 above)
Comment (k-y0u) [reply tests/resources/tf/bug78_emptyListAttribute.tf]: Done

[the remark this answers is not in the document]
Comment (k-y0u) [reply tests/resources/tf/bug78_emptyListAttribute.tf]: Done

**Review by justinehell — APPROVED** [review]

**Pull request merged**